tvm_client/src/net/server_link.rs

```diff
@@ -802,8 +805,8 @@ impl ServerLink {
         self.state.set_endpoint_addresses(endpoints).await;
     }
 
-    pub async fn get_addresses_for_sending(&self) -> Vec<String> {
-        self.state.get_addresses_for_sending().await
+    pub async fn _get_addresses_for_sending(&self) -> Vec<String> {
```

Comment:
Please do not add _ but put these functions under #[cfg(test)] if they are used only in tests